.github/workflows/update_combined_metadata.yaml

```diff
@@ -13,26 +13,45 @@ jobs:
     - name: Checkout curatedMetagenomicData
       uses: actions/checkout@v2
 
-    - name: Install remotes and curatedMetagenomicDataCuration
+    - name: Install curatedMetagenomicDataCuration
       run: |
         install.packages("remotes")
         remotes::install_github("waldronlab/omicidxClientR")
         remotes::install_github("waldronlab/curatedMetagenomicDataCuration")
       shell: Rscript {0}
 
     - name: Generate combined_metadata.rda
+      id: generate_metadata
       run: |
         library("dplyr")
         library("data.table")
```

Comment:
Just curious, what is data.table used for here? Is it a dependency of the `:::` function? At least for `::`, I would expect any imports or depends of those functions from another package to be imported into your namespace automatically as-needed.

Reply:
You're right. This is cruft from a different approach to load cMDC. I'll remove it.